source/includes/stackpath/_workloads.md

```diff
@@ -436,18 +439,20 @@ curl -X PUT \
   "deployments": [
     {
       "name": "toronto-1",
-      "pops": ["YYZ"],
+      "pops": [
+        "YYZ"
+      ],
       "enableAutoScaling": true,
-			"minInstancesPerPop": 1,
-			"maxInstancesPerPop": 2,
-			"cpuUtilization": 50
+      "minInstancesPerPop": 1,
+      "maxInstancesPerPop": 2,
+      "cpuUtilization": 50
     }
   ]
 }
 ```
 > Request body example for a CONTAINER Workload type:
 
-```json
+```js
```

Comment:
that must be an error

Reply:
When adding inline comments, I use `js` rather than `json` so the comments don't look like an error. Other than that, formatting remains the same in our docs.

| comments with `json` |comments with `js`|
| -- | --- |
| ![5-json-formatting](https://user-images.githubusercontent.com/8960233/109049652-800e4680-76a6-11eb-846b-e2dc57df8df8.jpg) | ![5-js-formatting](https://user-images.githubusercontent.com/8960233/109049688-88ff1800-76a6-11eb-85ca-b6197d52d4a2.jpg)|

Reply:
ok. Thanks for the clarification.